reco_utils/evaluation/spark_diversity_evaluator.py

```diff
@@ -0,0 +1,362 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from pyspark.sql.types import *
+from pyspark.sql import functions as F
+
+
+class DiversityEvaluator:
+    def __init__(
+        self,
+        train_df,
+        reco_df,
+        user_col="UserId",
+        item_col="ItemId",
```

Comment:
It would be good to follow the same naming as in the other classes:
```
class SparkRatingEvaluation:
    """Spark Rating Evaluator"""

    def __init__(
        self,
        rating_true,
        rating_pred,
        col_user=DEFAULT_USER_COL,
        col_item=DEFAULT_ITEM_COL,
        col_rating=DEFAULT_RATING_COL,
        col_prediction=DEFAULT_PREDICTION_COL,
    ):

```

Reply:
addressed